cypress/e2e/ondemand/fasrc/metrics.cy.js

```diff
@@ -0,0 +1,45 @@
+import { NAVIGATION, loadHomepage, navigateToMetrics } from "../../../support/utils/navigation.js";
+import {changeProfile} from "../../../support/utils/profiles";
+
+describe('FASRC Dashboard - Metrics Widget and Page', () => {
+  const fasrcClusterProfile = Cypress.env('fasrc_cluster_profile')
+  Cypress.config('baseUrl', NAVIGATION.baseUrl);
+  const cluster = Cypress.env('cluster')
+  const clusterName = cluster == 'cannon' ? 'Cannon' : cluster == 'fasse' ? 'FASSE' : 'Example'
```

Comment:
ClusterName should be configured in the environment specific properties file.
Similar to the welcome message:
https://github.com/hmdc/ondemand_testing/blob/main/ondemand/cypress.env.json.prod-cannon#L8